test/modules/CloudToDeviceMessageTester/Receiver.cs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace CloudToDeviceMessageTester
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Client;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil.TestResults;
+    using Microsoft.Extensions.Logging;
+    using TransportType = Microsoft.Azure.Devices.TransportType;
```

Comment:
Why are there duplication of `TransportType`?

Reply:
Need the full name since it conflicts with another TransportType somewhere else. This one is for deviceClient specifically which can only take AMQP